lib/rules/indent-function-args.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * @fileoverview Check for function args proper indentation.
+ * @author uberVU
+ * @copyright 2015 uberVU. All rights reserved.
+ * See LICENSE file in root directory for full license.
+ */
+'use strict';
+var getNodeIndent = require('../get-node-indent.js');
+var indentSize = 4;
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = function(context) {
+
+  var allignedWithParen = {
```

Comment:
By making them accept the object I meant defining it on their interface, which btw you can document using the [@interface](http://usejsdoc.org/tags-interface.html) tag. If the function doesn't need it then it can just not define it in its list of params.